doc/api/resources/orders.rst

```diff
@@ -104,6 +104,7 @@ url                                   string                     The full URL to
 payments                              list of objects            List of payment processes (see below)
 refunds                               list of objects            List of refund processes (see below)
 last_modified                         datetime                   Last modification of this object
+cancellation_date                     datetime                   Time of order cancellation (or ``null``)
```

Comment:
Maybe add a note that this will not be set for partial cancellations and is not reliable for orders canceled, reactivated and canceled again?